README.md

```diff
@@ -106,7 +106,7 @@ To integrate Stream Chat into your Xcode project using Carthage, specify it in y
 github "GetStream/stream-chat-swift"
 ```
 
-Then run: `carthage update --platform iOS --new-resolver`. This will build frameworks: `StreamChatClient.framework`, `StreamChatCore.framework` and `StreamChat.framework` into `<Path to your Project>/Carthage/Build/iOS/` from where you can add them to your project and link them with your app target. Follow with these steps:
```

Comment:
We need `carthage update` for fresh installs as outlined in https://github.com/Carthage/Carthage

Reply:
Oh, you're right. I didn't realize this about the installation from the user's point of view, not for us.